sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/collect.scala

```diff
@@ -265,3 +271,239 @@ private[aggregate] object CollectTopK {
     case _ => throw QueryCompilationErrors.invalidNumParameter(e)
   }
 }
+
+@ExpressionDescription(
+  usage = "_FUNC_(expr) - Returns the concatenation of non-null input values," +
```

Comment:
Could you describe arg: what the expression accepts and what returns. Please, add `arguments`

Reply:
added `arguments` + some minor improvements